package.json

```diff
@@ -46,12 +49,14 @@
   "devDependencies": {
     "event-stream": "^4.0.0",
     "jshint": "^2.5.6",
+    "nodemon": "^2.0.4",
     "pelias-mock-logger": "^1.1.0",
     "precommit-hook": "^3.0.0",
     "proxyquire": "^2.0.0",
     "tap-dot": "^2.0.0",
     "tape": "^4.2.2",
-    "temp": "^0.9.0"
+    "temp": "^0.9.0",
+    "winston": "^3.3.3"
```

Comment:
Quick one, having a look around, I don't see winston being used anywhere. Is it needed?

Thanks

Reply:
Not needed now I was using pelias-logger and it was needed for that but took it out in the end so I've removed it there now. Cheers.